error.go

```diff
@@ -25,9 +25,11 @@ type Err struct {
 	// previous holds the previous error in the error stack, if any.
 	previous error
 
-	// file and line hold the source code location where the error was
-	// created.
-	file string
```

Comment:
Can we keep file as well? Yes, the absolute path will be used, but it's still useful right?

Reply:
Not really. In a development build on your local machine yeah it's useful but as soon as we cut a release people would get values that make no sense with respect to their current system. Also the package name is the path relative to the source tree and you know where your source tree lives.